source/js/components/join/join.jsx

```diff
@@ -54,7 +54,6 @@ export default class JoinUs extends React.Component {
 
     let signupState = classNames({
       'row': true,
-      'py-3': true,
```

Comment:
Full page, full width (12 column) and half width (6 column). There's a set of styles for each in the scss file. It may be easiest to move this padding into the appropriate selectors in that file.